ALiAS_FAQs.md

```diff
@@ -0,0 +1,33 @@
+FAQs
+Q1. What is ASET AliAS?
+Ans. ALiAS is a community, founded in 2010, which was developed to promote the use of Linux in the
+campus and get students familiar with the OS.
```

Comment:
We don't do just OS anymore.

Reply:
There are alot of activities, lot of programs, lot of groups for alot of things that ALiAS does. Try to find a better, more innovative answer. That begins like `It's not your regular Amity Club`